src/tests/class_tests/openms/source/AAIndex_test.cpp

```diff
@@ -36,271 +36,34 @@
 #include <OpenMS/test_config.h>
 #include <OpenMS/CHEMISTRY/AASequence.h>
 #include <OpenMS/CHEMISTRY/AAIndex.h>
+#include <OpenMS/FORMAT/Base64.h>
 
 using namespace OpenMS;
 using namespace std;
 
 ///////////////////////////
 
-AASequence seq1 = AASequence::fromString("ALEGDEK");
-AASequence seq2 = AASequence::fromString("GTVVTGR");
-AASequence seq3 = AASequence::fromString("EHVLLAR");
+int main(){
 
+  std::vector<String> vec;
 
-START_TEST(AASequenceIndeces, "$Id$")
+  vec.resize(3);
+  vec[0] = "abcdefghijklmnopqrstuvwxyz";
+  vec[1] = "ABCDEFGHIJKLMNOPQRSTUVWXYZZ";
+  vec[2] = "1234567890";
 
-/////////////////////////////////////////////////////////////
-/////////////////////////////////////////////////////////////
+  String ausgabe;
 
-//sequence spec_id protein_id mass GB500 arginin_count KHAG800101 VASM830103 NADH010106 NADH010107 WILM950102 ROBB760107 OOBM850104 FAUJ880111 FINA770101 ARGP820102 M F H Q Y target_log
-//ALEGDEK 15 0587  761.368 1337.53 0  129.3 1.145   31  565  1.5200000 -6.60000e+00  -3.240000 1  7.18  5.23 0 0 0 0 0 2.08623342
-//GTVVTGR 15 0587  689.394 1442.70 1  383.2 1.042  241  403  7.1800000 -3.00000e-01 -16.010000 1  5.55  5.02 0 0 0 0 0 1.35346120
-//EHVLLAR 15 0587  837.494 1442.70 1  318.5 1.259  171  190 18.1300000  3.00000e-01  -9.970000 2  7.73  9.34 0 0 1 0 0 5.22075034
+  Base64 codingmachine;
 
-TOLERANCE_ABSOLUTE(0.01)
+  codingmachine.encodeStrings(vec, ausgabe, true , true);
 
-START_SECTION(static double calculateGB(const AASequence& seq, double T=500.0) )
-  TEST_REAL_SIMILAR(AAIndex::calculateGB(seq1), 1337.53)
-  TEST_REAL_SIMILAR(AAIndex::calculateGB(seq2), 1442.70)
-  TEST_REAL_SIMILAR(AAIndex::calculateGB(seq3), 1442.70)
+  std::cout << ausgabe << std::endl;
 
-  TEST_NOT_EQUAL(AAIndex::calculateGB(seq1,100.0), 1337.53)
-  TEST_NOT_EQUAL(AAIndex::calculateGB(seq2,100.0), 1442.70)
-  TEST_NOT_EQUAL(AAIndex::calculateGB(seq3,100.0), 1442.70)
-END_SECTION
+  codingmachine.encodeStrings(vec, ausgabe, false , true);
 
-START_SECTION(static double aliphatic(char aa))
-  TEST_REAL_SIMILAR(AAIndex::aliphatic('A'),1.0)
-  TEST_REAL_SIMILAR(AAIndex::aliphatic('B'),0.0)
-END_SECTION
+  std::cout << ausgabe << std::endl;
 
-START_SECTION(static double acidic(char aa))
-  TEST_REAL_SIMILAR(AAIndex::acidic('D'),1.0)
-  TEST_REAL_SIMILAR(AAIndex::acidic('A'),0.0)
-END_SECTION
+// should be : "eJxLTEpOSU1Lz8jMys7JzcsvKCwqLiktK6+orGJwdHJ2cXVz9/D08vbx9fMPCAwKDgkNC4+IjIpiMDQyNjE1M7ewNGAAACpTFWY="
```

Comment:
you delete so much code and comments, can you provide more Info
Also are you still testing the same things?